src/EventExport/Format/HTML/Properties/Footer.php

```diff
@@ -4,8 +4,17 @@
 
 namespace CultuurNet\UDB3\EventExport\Format\HTML\Properties;
 
-use CultuurNet\UDB3\StringLiteral;
+use CultuurNet\UDB3\Model\ValueObject\String\Behaviour\IsString;
+use CultuurNet\UDB3\Model\ValueObject\String\Behaviour\Trims;
 
-class Footer extends StringLiteral
+class Footer
```

Comment:
These & the other classes in the `Format/HTML/Properties`-Folder could probably made final

Reply:
done